locales-uk-UA.xml

```diff
@@ -16,50 +16,41 @@
     <date-part name="year"/>
   </date>
   <terms>
-    <term name="accessed">accessed</term>
+    <term name="accessed">дата звернення</term>
     <term name="and">і</term>
     <term name="and others">та інші</term>
     <term name="anonymous">анонімний</term>
     <term name="anonymous" form="short">анон.</term>
     <term name="at">на</term>
-    <term name="available at">available at</term>
-    <term name="by">by</term>
-    <term name="circa">circa</term>
+    <term name="available at">доступний у</term>
+    <term name="by">відповідно до</term>
+    <term name="circa">близько</term>
     <term name="circa" form="short">c.</term>
-    <term name="cited">cited</term>
-    <term name="edition">
-      <single>edition</single>
-      <multiple>editions</multiple>
-    </term>
-    <term name="edition" form="short">ed</term>
+    <term name="cited">цит. за</term>
+    <term name="edition">видання</term>
+    <term name="edition" form="short">вид.</term>
     <term name="et-al">et al.</term>
-    <term name="forthcoming">forthcoming</term>
+    <term name="forthcoming">майбутній</term>
     <term name="from">із</term>
-    <term name="ibid">ibid.</term>
+    <term name="ibid">там само</term>
     <term name="in">в</term>
     <term name="in press">у пресі</term>
     <term name="internet">інтернет</term>
     <term name="interview">інтервю</term>
     <term name="letter">лист</term>
-    <term name="no date">no date</term>
-    <term name="no date" form="short">n.d.</term>
+    <term name="no date">без дати</term>
+    <term name="no date" form="short">б. д.</term>
     <term name="online">online</term>
-    <term name="presented at">presented at the</term>
-    <term name="reference">
-      <single>reference</single>
-      <multiple>references</multiple>
-    </term>
-    <term name="reference" form="short">
-      <single>ref.</single>
-      <multiple>refs.</multiple>
-    </term>
-    <term name="retrieved">retrieved</term>
-    <term name="scale">scale</term>
-    <term name="version">version</term>
+    <term name="presented at">представлена на</term>
+    <term name="reference">список використаних джерел</term>
+    <term name="reference">примітки</term>
```

Comment:
@vkochyn, you have two translation for `<term name="reference"` here. Is the second one the "short" form?